src/RevCPU.cc

```diff
@@ -2546,6 +2546,7 @@ void RevCPU::InitThread(std::shared_ptr<RevThread>& ThreadToInit){
 
 void RevCPU::AssignThread(uint32_t ThreadID, uint32_t ProcID){
   output.verbose(CALL_INFO, 4, 0, "Assigning Thread %" PRIu32 " to Processor %" PRIu32 "\n", ThreadID, ProcID);
+  // TODO: (BEFORE MERGE): Lee, do I need to make this an auto&
```

Comment:
No, since you're only using it as a value, as a pointer in `Thread->`, and as a non-reference parameter in `... .emplace(ThreadID, Thread);`